template.tpl

```diff
@@ -96,6 +96,85 @@ ___TEMPLATE_PARAMETERS___
     "name": "eventType",
     "type": "SELECT"
   },
+  {
+    "type": "GROUP",
+    "name": "eventMetadataGroup",
+    "displayName": "Additional Event Information",
+    "groupStyle": "NO_ZIPPY",
+    "subParams": [
+      {
+        "type": "SIMPLE_TABLE",
+        "name": "eventMetadataParams",
+        "displayName": "",
+        "simpleTableColumns": [
+          {
+            "defaultValue": "",
+            "displayName": "Parameter Name",
+            "name": "name",
+            "type": "SELECT",
+            "selectItems": [
+              {
+                "value": "itemCount",
+                "displayValue": "Item Count"
+              },
+              {
+                "value": "value",
+                "displayValue": "Event Value"
```

Comment:
Maybe "Transaction Value" to match "Transaction ID"?

More generally -- even if you keep the name -- what about adding a parenthetical expression explaining they should use the smallest subdivision of the currency here?  I know detailed instructions come separately, but as a remember-the-human sorta thing, IMHO it's nice to have a friendly reminder that this field is not the currency base unit.  Maybe "Transaction value (pennies, paise, cents, satoshis ...)"?

Reply:
Definitely for the first. For the second, I really wanted to give a little tooltip, but you can't put menu-item-level tooltips. What about `Transaction Value (in smallest subunit of currency)` versus listing out some options?

(TIL USD denomination starts at $1 and not $0.01)